teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1317 @@
+{
```

Comment:
<p>use openmm unit instead of parmed unit if applicable</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>